docs/data/sdks/sdk-overview.md

```diff
@@ -11,6 +11,8 @@ Browse all Amplitude's current SDK offerings. Maintenance SDKs are listed in the
 
 ## Analytics SDKs
 
+Please check the comparison section in the migration doc for the differences between SDKs for each platform.
```

Comment:
The comparsion section is removed now right? Maybe we should provide links to the individual platform comparisons here instead.

Reply:
I mean to go to the individual platform to check the comparison section. I will refactor this sentence.

Reply:
I finally decided to delete this part.